lib/cpp/src/thrift/TApplicationException.h

```diff
@@ -21,6 +21,7 @@
 #define _THRIFT_TAPPLICATIONEXCEPTION_H_ 1
 
 #include <thrift/Thrift.h>
+#include <stdint.h>
```

Comment:
As I said, for windows it's OK, for non-windows without stdint.h, it will fail.
We could improve config.hin and build/cmake/config.h.in to do the same thing as Windows when boost is avaiable.